docker-compose.yml

```diff
@@ -24,3 +24,11 @@ services:
     environment:
       - GRAPH_API_HOST=host.docker.internal
       - GRAPH_API_PORT=18080
+      - ONTOLOGY_API_HOST=host.docker.internal
+      - ONTOLOGY_API_PORT=18082
+  ontology_api:
+    build: ontology_api
+    ports:
+      - "18082:80"
+    extra_hosts: 
+      - "host.docker.internal:host-gateway"
```

Comment:
For the purpose of the testing, we need another container of the `GriseraAPI`, that will run with the `OntologyAPI`. Within this container, there should be passed some variable, lets call it `PERSISTENCE_TYPE`, which value for OntologyAPI will be set to `2`. Please add:
```
  grisera_api_ontology:
    build: grisera_api_ontology
    ports:
      - "18083:80"
    extra_hosts: 
      - "host.docker.internal:host-gateway"
    environment:
      - ONTOLOGY_API_HOST=host.docker.internal
      - ONTOLOGY_API_PORT=18082
      - PERSISTENCE_TYPE=2
```

Reply:
To be clear, now we will have 5 containers:
- `grisera_neo4j` - Neo4J graph database
- `graph_api` - backend bridge between `grisera_api` and graph database
- `ontology_api`- backend that allows to process ontology file
- `grisera_api` - REST API working with the graph database
- `grisera_api_ontology` - REST API working with the ontology backend

Both `grisera_api` and `grisera_api_ontology` will be publicly available.

Reply:
gotowe

Reply:
OK.